Doc/library/datetime.rst

```diff
@@ -804,6 +811,11 @@ Other constructors, all class methods:
    except the latter formula always supports the full years range: between
    :const:`MINYEAR` and :const:`MAXYEAR` inclusive.
 
+   .. warning::
+
+      Because naive datetimes are treated by many functions as local times.
```

Comment:
Can you change this wording to be of a similar sentence structure as in the earlier section (for `datetime.utcnow()`)?